nixos/modules/services/monitoring/prometheus/exporters/rtl_433.nix

```diff
@@ -0,0 +1,84 @@
+{ config, lib, pkgs, options }:
+
+with lib;
+
+let
+  cfg = config.services.prometheus.exporters.rtl_433;
+in
+{
+  port = 9550;
+
+  extraOpts = let
+    nameLocation = with types; {
+      name = mkOption {
+        type = str;
+        description = "Name to match.";
+      };
+      location = mkOption {
+        type = str;
+        description = "Location to match.";
+      };
+    };
+  in
+  {
+    flags = mkOption {
```

Comment:
I would rename to e.g. rtl433Flags to avoid confusion with extraFlags.

Reply:
Done.